api/learn/preview.go

```diff
@@ -58,14 +48,14 @@ func (api *APIClient) PollForBuildResponse(releaseID int, attempts *uint8) (*Pre
 
 	if p.Status == "processing" || p.Status == "pending" {
 		*attempts--
+		time.Sleep(2 * time.Second)
 
-		if *attempts <= uint8(0) {
+		if *attempts == uint8(0) {
```

Comment:
I ran into an issue where if you pass 0 into this function it breaks forever while testing. I think it might be better defensive coding to keep this a `<=` and to keep the time.Sleep after the attempts check